networkstatsquerier/main.go

```diff
@@ -27,63 +28,108 @@ var totalUptimeChecksCounterVec *prometheus.CounterVec
 var totalFailedUptimeChecksCounterVec *prometheus.CounterVec
 
 // getHTTPRequestTiming records the http request characteristics for the endpoint specified by urlString
-// Record the metrics into the appropriate prometheus register under the label specified by organizationName
-func getHTTPRequestTiming(urlString string, organizationName string) {
-	ctx := context.Background()
-	// Closing context if HTTP request and response processing is not completed within 30 seconds.
-	// This includes dropping the request connection if there's no reply within 30 seconds.
-	ctx, cancelFunc := context.WithDeadline(ctx, time.Now().Add(30*time.Second))
-	defer cancelFunc()
+func getHTTPRequestTiming(message []byte, args *map[string]interface{}) error {
+	// Get arguments
+	wkrs, ok := (*args)["workers"].(*workers.Workers)
```

Comment:
use an object to simplify and reduce possible errors?

so args would just have one argument that is that object, and that object would be populated with all of this data.